src/sdfv.ts

```diff
@@ -50,7 +50,7 @@ export class SDFV {
     // Points-per-pixel threshold for not drawing memlets/interstate edges.
     public static EDGE_LOD: number = 8;
     // Points-per-pixel threshold for not drawing node shapes and labels.
-    public static NODE_LOD: number = 60;
+    public static NODE_LOD: number = 4.0;
```

Comment:
This changed quite a bit. Does it mean anything different now?

Reply:
This was a mistake, seems that I changed it to test something but forgot to change it back. On that note: the renderer seems incredibly robust to this value change... I did not immediately observe different behavior between the two values. Strange, but irrelevant at this point.